### ISSUE TITLE
Plugin does not hide ad display container if adtimeout event occurs.

### ISSUE THREAD

**Issue by alexbakum** [googleads/videojs-ima#79]
Hello.

I've noticed an issue in the case where DFP does return an empty response.
The thing is when DFP returns an empty response the plugin destroys ads manager and resumes content playback but does not hide ads display container so video and controls are not clickable any more.

Do you have any suggestions how to set up a workaround on this?

**Comment by shawnbuso** [comment]
Sorry, didn't mean to click "close". Looking into this now :)

**Comment by shawnbuso** [comment]
Do you have a page where we can see this issue in action? I tested it out by requesting example.com as my ad tag, which causes an error, but the controls still work fine for me. Is this a problem with a particular browser or OS?

**Comment by alexbakum** [comment]
Hello. The issue was not correlating with any browser or OS last time (yesterday evening at my tz) I've faced this bug, the behaviour is the same everywhere.

I don't have the test page for now. But I've tested your patch on my pc and it seems to fix the issue.
Also I have a question: can ima sdk query for some new ad content while ad container is hidden? What behaviour will we experience if some content IMASDK fetches from DFP (some midroll or postroll for example)?
